deeplabcut/pose_estimation_pytorch/runners/snapshots.py

```diff
@@ -68,18 +75,21 @@ class TorchSnapshotManager:
                 "optimizer": optimizer.state_dict()
             })
     """
+
     task: Task
     model_folder: Path
     key_metric: str | None = None
     key_metric_asc: bool = True
     max_snapshots: int = 5
     save_epochs: int = 25
     save_optimizer_state: bool = False
+    _best_model_epochs: int = -1
+    _best_metric: float | None = None
+    _key: str = field(init=False)
```

Comment:
No docstring because these are "private", is that right?

Reply:
exactly 👍